scrnaseq_hto.Rmd

```diff
@@ -133,39 +229,49 @@ This section of the report shows how cells are assigned to their sample-of-origi
 
 ## Normalisation of HTO counts
 We start the analysis by normalising raw HTO counts. HTO counts for each cell are divided by the total counts for that cell and multiplied by 10,000. This is then natural-log transformed. 
-```{r hto_norm}
-sc = Seurat::NormalizeData(sc, assay="HTO", normalization.method="LogNormalize")
+```{r hto_normalisation}
+
+# HTO
```

Comment:
How about `param$norm` is "CLR" or "LogNormalize" and you skip the if else, and just call it with `param$norm`?

Reply:
Done